spec/reducers/wallet.spec.ts

```diff
@@ -60,7 +60,12 @@ describe('wallet reducer', () => {
   });
 
   it('should handle WALLET_SET_TOKEN_BALANCES', () => {
-    const tokenBalances = { OMG: TokenValue('20') };
+    const tokenBalances = {
```

Comment:
would be great to see a test for the error case

Reply:
ack